Gemfile.lock

```diff
@@ -174,4 +174,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   2.4.6
+   2.3.22
```

Comment:
It's a little odd that this reverted but I don't think there's anything to be concerned about. SHip it.